sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb

```diff
@@ -872,6 +872,79 @@
     "print(resp.text)"
    ]
   },
+  {
```

Comment:
Minor: Does this visualization work as expected. Just wanted to check since it is something that won't be caught in github actions.

Reply:
Yes, I checked it on local run.

Reply:
<img width="256" alt="Screenshot 2022-06-29 142840" src="https://user-images.githubusercontent.com/24417856/176396995-5ef28a2a-56a9-4b23-b984-7d062774e532.png">
<img width="260" alt="image" src="https://user-images.githubusercontent.com/24417856/176397128-f74a6e4a-b640-4a02-a765-22c28a91cf03.png">